src/App.jsx

```diff
@@ -14,70 +16,116 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const Good = ({ good }) => (
+    <tr
+      data-cy="Good"
+      className={cn({ 'has-background-success-light': selectedGood === good })}
+    >
+      <td>
+        {selectedGood === good ? (
+          <button
+            data-cy="RemoveButton"
+            type="button"
+            className="button is-info"
+            onClick={() => setSelectedGood('')}
```

Comment:
```suggestion
            onClick={clearGood}
```
It's better to create separate functions rather than creating an anonymous callback.